$OpenBSD: patch-glib_poppler_h,v 1.1 2014/04/28 16:48:27 kili Exp $

This typedef is already in poppler-structure-element.h, which is
#included by poppler.h.

--- glib/poppler.h.orig	Sun Feb  9 23:08:52 2014
+++ glib/poppler.h	Sun Apr 27 12:43:33 2014
@@ -210,7 +210,6 @@ typedef struct _PopplerAnnotSquare         PopplerAnno
 typedef struct _PopplerQuadrilateral       PopplerQuadrilateral;
 typedef struct _PopplerStructureElement    PopplerStructureElement;
 typedef struct _PopplerStructureElementIter PopplerStructureElementIter;
-typedef struct _PopplerTextSpan            PopplerTextSpan;
 
 typedef enum
 {
